src/index.html

```diff
@@ -9,6 +9,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="containerLogo">
+        <a href="#home" class="linkLogo">
+          <img
+            class="logo"
+            src="../src/images/logo.png"
+            alt="Store logo MOYO"
+          >
+        </a>
+      </div>
+
+      <nav class="navBlock">
+
+        <ul class="ulLink">
```

Comment:
```suggestion
      <nav class="navBlock">
        <ul class="ulLink">
```
Don't add new line between parent and child elements